images/20-rootfs/Dockerfile

```diff
@@ -82,6 +82,9 @@ RUN rm -rf \
 COPY --from=k3s /output/install.sh /usr/src/image/libexec/k3os/k3s-install.sh
 COPY --from=progs /output/metadata /usr/src/image/sbin/metadata
 COPY --from=progs /output/kubectx/kubectx /output/kubectx/kubens /usr/src/image/bin/
+ARG ARCH
```

Comment:
Is this necessary?

Reply:
I've run a check and as ARCH is already defined this can be removed. I'll remove and we can check that this builds correctly.

Reply:
PR Updated and CI checks passed.